doc/rra/rra.tex

```diff
@@ -0,0 +1,115 @@
+\documentclass[runningheads]{llncs}
+%
+\usepackage{graphicx}
+\usepackage{float}
+\usepackage{amssymb}
+\usepackage{amsmath}
+\usepackage{listings}
+\usepackage{color}
+\usepackage{xcolor}
+\usepackage{booktabs}
+\usepackage{tabularx}
+\usepackage{cite}
+
+\begin{document}
+%
+\title{Unbounding CBMC using Replication Reducing Abstraction}
```

Comment:
Thanks for adding the paper.  I think the development team probably need to have a discussion over whether this is the best form of this information as it adds dependencies (LaTeX) and could be harder to maintain.